generators/express/templates/base/package.json.ejs

```diff
@@ -9,17 +9,22 @@
     "@thetribe/eslint-config-typescript": "^0.3.0",
     "@tsconfig/node14": "^1.0.0",
     "@types/express": "^4.17.9",
+    "@types/jest": "^26.0.23",
     "@typescript-eslint/eslint-plugin": "^4.14.1",
     "@typescript-eslint/parser": "^4.14.1",
     "eslint": "^7.18.0",
     "eslint-plugin-import": "^2.22.1",
+    "jest": "^27.0.6",
     "nodemon": "^2.0.7",
+    "ts-jest": "^27.0.3",
     "ts-node": "^9.1.1",
     "typescript": "^4.1.2"
   },
   "scripts": {
     "build": "tsc",
     "lint": "eslint src --ext ts,tsx",
-    "start": "nodemon --ext js,ts --watch src --exec ts-node src"
+    "start": "nodemon --ext js,ts --watch src --exec ts-node src",
+    "test": "jest",
+    "test:coverage": "yarn test --coverage"
```

Comment:
It could be nice to add a script for jest with watch mode :
`"test:watch": "yarn test --watch"`

Reply:
Well you can use `yarn test --watch` directly, I'm not sure adding the script doesn't add that much since it's only 2 character less to type.

Reply:
I agree with you but since the same thing can be done with coverage so i think it's more consistant in term of script proposed. Anyway that fine for me as it is :)